tst/EnergyPlus/unit/PoweredInductionUnits.unit.cc

```diff
@@ -523,3 +523,55 @@ TEST_F(EnergyPlusFixture, SeriesPIUTest1)
     DataZoneEnergyDemands::ZoneSysEnergyDemand.deallocate();
     DataZoneEnergyDemands::CurDeadBandOrSetback.deallocate();
 }
+
+
+// cf: https://github.com/NREL/EnergyPlus/issues/7183
+TEST_F(EnergyPlusFixture, PIUArrayOutOfBounds) {
```

Comment:
Good, succinct unit test.  Don't really need the comment above stating what it is for, we can track that through our history, but no point in re-running CI to remove the comment.